owncloudData/src/main/java/com/owncloud/android/data/files/datasources/LocalFileDataSource.kt

```diff
@@ -21,10 +21,11 @@
 package com.owncloud.android.data.files.datasources
 
 import androidx.lifecycle.LiveData
+import com.owncloud.android.domain.availableoffline.model.AvailableOfflineStatus
 import com.owncloud.android.domain.files.model.OCFile
 
 interface LocalFileDataSource {
-    fun copyFile(sourceFile: OCFile, targetFile: OCFile, finalRemotePath: String, remoteId: String)
+    fun copyFile(sourceFile: OCFile, targetFolder: OCFile, finalRemotePath: String, remoteId: String)
```

Comment:
In the second parameter of this method, do we specify the generated file that will result from the copy or the destination folder of the copy? I see you updated the name but the implementation continues being the same. If it is the second option of the ones previously mentioned, GREAT!! 🏅

Reply:
`targetFolder` is the destination folder, probably `destinationFolder` would be a better name. I updated just the name because I think that it is easier to understand the logic, but it keeps the same behavior.